#! /bin/sh -e
## 40imagepath.dpatch by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
##
## DP: Correct path to logo and images

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch)   ( cd BUILD/sql-ledger && patch $patch_opts -p1 ) < $0 ;;
       -unpatch) ( cd BUILD/sql-ledger && patch $patch_opts -p1 -R ) < $0 ;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0
@DPATCH@
diff -ur sql-ledger-2.4.5/bin/mozilla/am.pl BUILD/sql-ledger/bin/mozilla/am.pl
--- sql-ledger-2.4.5/bin/mozilla/am.pl	Fri Nov  5 04:53:55 2004
+++ sql-ledger/bin/mozilla/am.pl	Mon Nov 29 20:37:08 2004
@@ -2349,7 +2349,7 @@
 
 </pre>
 <center>
-<a href="http://www.sql-ledger.org" target=_top><img src=sql-ledger.gif border=0></a>
+<a href="http://www.sql-ledger.org" target=_top><img src="images/sql-ledger.png" border=0></a>
 <h1 class=login>|.$locale->text('Version').qq| $form->{version}</h1>
 
 <p>
diff -ur sql-ledger/bin/mozilla/admin.pl BUILD/sql-ledger/bin/mozilla/admin.pl
--- sql-ledger-2.4.5/bin/mozilla/admin.pl	Tue Nov 23 22:56:46 2004
+++ sql-ledger/bin/mozilla/admin.pl	Sat Nov 27 21:35:55 2004
@@ -104,7 +104,7 @@
 
 <div align=center>
 
-<a href="http://www.sql-ledger.org"><img src=sql-ledger.png border=0>&trade;</a>
+<a href="http://www.sql-ledger.org"><img src="images/sql-ledger.png" border=0>&trade;</a>
 <h1 class=login>|.$locale->text('Version').qq| $form->{version}<p>|.$locale->text('Administration').qq|</h1>
 
 <form method=post action="$form->{script}" name=admin>
diff -ur sql-ledger/bin/mozilla/login.pl BUILD/sql-ledger/bin/mozilla/login.pl
--- sql-ledger-2.4.5/bin/mozilla/login.pl	Fri Nov  5 05:15:38 2004
+++ sql-ledger/bin/mozilla/login.pl	Sat Nov 27 21:24:48 2004
@@ -109,7 +109,7 @@
 <center>
 <table class=login border=3 cellpadding=20>
   <tr>
-    <td class=login align=center><a href="http://www.sql-ledger.org" target=_top><img src=sql-ledger.gif border=0>&trade;</a>
+    <td class="login" align="center"><a href="http://www.sql-ledger.org" target=_top><img src="images/sql-ledger.png" border="0">&trade;</a>
 <h1 class=login align=center>|.$locale->text('Version').qq| $form->{version}
 </h1>
 
@@ -170,7 +170,7 @@
 <center>
 <table class=login border=3 cellpadding=20>
   <tr>
-    <td class=login align=center><a href="http://www.sql-ledger.org" target=_top><img src=sql-ledger.gif border=0>&trade;</a>
+    <td class="login" align="center"><a href="http://www.sql-ledger.org" target=_top><img src="images/sql-ledger.png" border="0">&trade;</a>
 <h1 class=login align=center>|.$locale->text('Version').qq| $form->{version}
 </h1>
 
